planet/config.ini

```diff
@@ -65,6 +65,9 @@ name = Kestrel Computer Project
 [http://blog.verificationgentleman.com/feeds/posts/default]
 name = Verification Gentleman
 
+[http://abopen.com/blog/category/osddi/feed]
```

Comment:
We're trying to avoid adding more HTTP blogs to the feed, as they cause mixed-content warnings when they contain images. 

It seems the ssl certificate for abopen.com has expired on 05.01.2016, so some action is required.

Andrew, is HTTPS support on your todo list for the short term? Otherwise I'm going to add the blog with HTTP for now and then we can change it later on.